src/helpers.cpp

```diff
@@ -105,4 +108,18 @@ int wlcs::helpers::get_argc()
 char const** wlcs::helpers::get_argv()
 {
     return ::argv;
-}
\ No newline at end of file
+}
+
+void wlcs::helpers::set_module_under_test(char const* dso_name)
+{
+    ::dso_name = dso_name;
+}
+
+std::shared_ptr<WlcsServerIntegration const> wlcs::helpers::get_test_hooks()
+{
+    auto dso = std::make_shared<wlcs::SharedLibrary>(::dso_name);
+
+    auto entry_point = dso->load_function<WlcsServerIntegration const*>("entry_point");
```

Comment:
Shouldn't the entry point be named something more specific than "entry_point"?

At least let's namespace it with a "wlcs_" prefix.

Reply:
Renamed as `wlcs_server_integration`